tests/unit/test_exit_signal.py

```diff
@@ -0,0 +1,37 @@
+import subprocess
+import sys
+
+from tox._pytestplugin import create_files
+
+
+def test_exit_code_signal(cmd, tmpdir):
+    create_files(tmpdir, {
+        # TODO: can this be minified?
+        "tox.ini": """\
+            [tox]
+            skipsdist = true
+            skips_install = true
+
+            [testenv]
+            commands = {interpreter} tfile.py
+        """.format(
+            interpreter=sys.executable
+        ),
+        "tfile.py": """\
+            import os, signal
+            os.kill(os.getpid(), signal.SIGTERM)
+        """,
+    })
+    p = subprocess.Popen([
+        sys.executable, '-m', 'tox'
+    ], stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd=str(tmpdir))
+    p.wait()
+    out = p.stdout.read().decode()
+    assert p.returncode == 1
+    assert (
+        "ERROR: InvocationError for command {} tfile.py (exited with code -15 (SIGTERM))".format(
+            sys.executable
+        )
+        in out
+    )
+    assert p.stderr.read() == b""
```

Comment:
@asottile 
Is there a better pattern for this test?  (mainly with regard to speed)

Reply:
there's a pytest plugin that's used elsewhere -- it won't be much faster but in most cases that's ok

Reply:
Do you mean the "initproj" fixture?
(that got too much in the way)

I think tweaking tox.ini would be good to just run the command if possible.

Reply:
???

It should be as simple as:

```python
def test...(initproj, cmd):
    initproj(
        "pkg",
        filedefs={
            "tox.ini": """\
                [tox]
                skipsdist = true
                skips_install = true
                [testenv]
                commands = python tfile.py
            """,
            "tfile.py": """\
                import os, signal
                os.kill(os.getpid(), signal.SIGTERM)
            """,
        },
    )
    result = cmd("-e", "py")

    ...
```

Reply:
Yeah, but it's not.
Dropping the integration test.

(btw: should have been "skip_install")